gtfs/spec/en/reference.md

```diff
@@ -219,14 +219,14 @@ File: **Required**
+|  arrival_time | Time | **Conditionally required** | The **arrival_time** specifies the arrival time at a specific stop for a specific trip on a route. The time is measured from "noon minus 12h" (effectively midnight, except for days on which daylight savings time changes occur) at the beginning of the service day. For times occurring after midnight on the service day, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins. If you don't have separate times for arrival and departure at a stop, enter the same value for **arrival_time** and **departure_time**.<br/><br/>Scheduled stops where the vehicle strictly adheres to the specified arrival and departure times are timepoints. For example, if a transit vehicle arrives at a stop before the scheduled departure time, it will hold until the departure time. If this stop is not a timepoint, use either an empty string value for the **arrival_time** field or provide an interpolated time. Further, indicate that interpolated times are provided via the **timepoint** field with a value of zero. If interpolated times are indicated with **timepoint**=0, then time points must be indicated with a value of 1 for the **timepoint** field. This field is required for all stops that are time points and the first and last stop in a trip, otherwise it is optional.<br/><br/> Times must be eight digits in HH:MM:SS format (H:MM:SS is also accepted, if the hour begins with 0). Do not pad times with spaces. The following columns list stop times for a trip and the proper way to express those times in the **arrival_time** field: |  |
```

Comment:
This is somewhat related - I'd consider moving everything that describes when this field is required into a separate paragraph for clarity and ease of reading. At the moment, that's spread over two paragraphs, interspersed with other things.

Maybe something like this -

```
<br/><br/>This field is required for all stops that are time points and the first and last stop in a trip, otherwise it is optional. If you don't have separate times for arrival and departure at a stop, enter the same value for **arrival_time** and **departure_time**.<br/></br>
```

It may also be worth emphasising the requirement that `arrival_time` must be specified if `departure_time` is specified and vice-versa.

Reply:
I agree that the arrival_time description is not very readable and should be amend. However, that's something @antrim and I hope to address in a separate PR that will also make a number of other editorial changes.